### ISSUE TITLE
Release new version on PyPI?

### ISSUE THREAD

**Issue by manodeep** [qyx268/astrodatapy#15]
I was trying to use the `IMF_out` parameter in `number_density` but was getting an error. That made me realise that the PyPI version is older and does not track the latest version on GitHub. If you are okay with "releasing" a new version on GitHub, then you can use this [automated action](https://github.com/manodeep/Corrfunc/blob/master/.github/workflows/pypi-publish.yml) to upload your latest release to PyPI. I am happy to add that GH action as a PR, if that would be useful to you (but you will still need to release the package on GH)

**Comment by qyx268** [comment]
Yes, definitely! I've just released the package on Github. v0.2 is the latest one with all the udpates.
